src/main/java/com/krystian/ThirdTask.java

```diff
@@ -0,0 +1,91 @@
+package com.krystian;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.*;
+
+public class ThirdTask {
+    private List<String> listOfStrings = new ArrayList<>();
+
+    public void pareAllElementsToInteger(String path) throws IOException {
+        listOfStrings = Files.readAllLines(Paths.get(path));
+        System.out.println(listOfStrings.size());
+        int counter = 0;
+        for(int x = 0; x < listOfStrings.size(); x++){
+            String [] charsOfString = listOfStrings.get(x).split("");
+            String [] firstPart = Arrays.copyOfRange(charsOfString, 0, (charsOfString.length + 1)/2);
+            String [] secondPart = Arrays.copyOfRange(charsOfString, (charsOfString.length + 1)/2, charsOfString.length);
+            counter = counter + getPointForLetter(returnCommonChar(firstPart, secondPart));
+        }
+        System.out.println(counter);
+    }
+
+
+    private String returnCommonChar(String [] first,String [] second){
+        boolean stop = false;
+        String common = "";
+        for(int x = 0; x < first.length; x++){
+            for(int y = 0; y < second.length; y++){
+                if(first[x].equals(second[y])) {
+                    common = first[x];
+                    stop = true;
+                    break;
+                }
+            }
+            if(stop){
```

Comment:
ten if to tez zbedny, bo inicjalizujesz stop na false. Wiec jezeli wejdzie kiedykolwiek w ifa ktory go ustawia i leci break, ktory konczy fora nieodwolalnie to dalszy kod nie ma sensu w tym kontekscie